lib/clients/GcpClient.py

```diff
@@ -0,0 +1,553 @@
+from .BaseClient import BaseClient
+from googleapiclient import discovery
+from google.oauth2 import service_account
+from google.cloud import storage
+from google.cloud.storage import Blob
+from .BaseClient import BaseClient
+from ..models.Snapshot import Snapshot
+from ..models.Volume import Volume
+from ..models.Attachment import Attachment
+import json
+import glob
+
+class GcpClient(BaseClient):
+    def __init__(self, operation_name, configuration, directory_persistent, directory_work_list, poll_delay_time,
+                 poll_maximum_time):
+        super(GcpClient, self).__init__(operation_name, configuration, directory_persistent, directory_work_list,
```

Comment:
`super().method()` also does the same thing as `super(C, self).method(arg)`. Since in other IaaS clients as well, it is specified in the same way, it will be better to keep it same way everywhere.

Reply:
ok